distribution/server-x-dist/pom.xml

```diff
@@ -49,7 +49,7 @@
     </dependencies>
 
     <build>
-        <finalName>keycloak.x-${project.version}</finalName>
+        <finalName>keycloak-${project.version}</finalName>
```

Comment:
Could you please also update these resources to reflect the name changes:

* https://github.com/keycloak/keycloak/blob/6c12c188542aeacc6d40aff9160e83b6a12f5b97/quarkus/tests/integration/src/main/java/org/keycloak/it/utils/DockerKeycloakDistribution.java#L33
* https://github.com/keycloak/keycloak/blob/781ceb24fddec2655e3461f1b297aac02c737f04/quarkus/tests/integration/src/main/java/org/keycloak/it/utils/RawKeycloakDistribution.java#L253

We were using Maven API to resolve the artifact but we gave up in favor of referencing it directly from the filesystem. We should be improving that.

These changes should help with the quarkus tests.

Reply:
Done